src/Pillar-ExporterCore/PRTarget.class.st

```diff
@@ -8,15 +8,27 @@ Class {
 	#category : #'Pillar-ExporterCore-Base'
 }
 
+{ #category : #'as yet unclassified' }
+PRTarget class >> associateTarget: aFormat [
+	^ self allSubclasses
+		select: [ :each | (each isAbstract not) and: each builderName = aFormat asString ]
+]
```

Comment:
Maybe this method could be called "targetForName:"? or something like that?

Reply:
Yes, it's solved. I replaced associateTarget: by targetForName: